dev/packages/example/endpoint-0.0.1/manifest.yml

```diff
@@ -11,6 +11,19 @@ release: beta
 type: solution
 license: basic
 
+
```

Comment:
@james-elastic I start to challenge if the package itself is a solution or an integrations package and the difference is only below in the datasource.